File: Examples_VI_CtCI_C/Example 5/Example5.c
#include <stdio.h>
// 0 (ab) runtime.

int main() {
	int arrayA[] = {1, 2, 3, 4, 5};
	int arrayB[] = {1, 2, 3, 4, 5};
	int lenA = sizeof(arrayA)/sizeof(arrayA[0]);
	int lenB = sizeof(arrayB)/sizeof(arrayB[0]);
	int i, j, k;
	
	for (i = 0; i < lenA; i++) {
		for (j = 0; j < lenB; j++) {
			for (k = 0; k < 100000; k++) {
				printf("%d, %d\n", arrayA[i], arrayB[j]);
				}
		}
	}
	return 0;
}

File: Examples_VI_CtCI_Java/src/Example5.java

public class Example5 {
	// 0 (ab) runtime.
	
	static void printUnorderedPairs (int[] arrayA, int[] arrayB) {
		for (int i = 0; i < arrayA.length; i++) {
			for (int j = 0; j < arrayB.length; j++) {
				for (int k = 0; k < 100000; k++) {
					System.out.println(arrayA[i] + ", " + arrayB[j]);
				}
			}
		}
	}
	
	public static void main(String[] args) {
		int[] arrayA = {1, 2, 3, 4, 5};
		int[] arrayB = {1, 2, 3, 4, 5};
		
		printUnorderedPairs(arrayA, arrayB);
	}
}
